skhep/math/vectors.py

```diff
@@ -1068,24 +1008,24 @@ def deltar(self, other):
         return sqrt( self.deltaeta(other)**2 + self.deltaphi(other)**2 )
 
     def isspacelike(self):
-        """Check if Lorentz vector is space-like."""
+        """Check if LorentzVector is space-like."""
```

Comment:
Again, why give the class name? The "space-likeliness" and other is a property of "Lorentz vectors". You would not be using "LV" had the class been called "LV", would you ;-). These occurrences are better as "Lorentz vector" I reckon.

Reply:
Hahaha, yes there was a lot of different names at some stages so I made everything uniform but yeah I agree to change that!